cg/meta/upload/vogue.py

```diff
@@ -0,0 +1,22 @@
+from cg.apps.gt import GenotypeAPI
+from cg.apps.vogue import VogueAPI
+import json
+from django.core.serializers.json import DjangoJSONEncoder
```

Comment:
eh .. could we use something less dramatic than django?

Reply:
Ops. I wasn't using that anymore. Removing the import.